cli/azd/pkg/project/scaffold_gen.go

```diff
@@ -225,14 +225,42 @@ func infraSpec(projectConfig *ProjectConfig) (*scaffold.InfraSpec, error) {
 	}
 
 	// create reverse frontends -> backends mapping
-	for _, svc := range infraSpec.Services {
+	for i := range infraSpec.Services {
+		svc := &infraSpec.Services[i]
```

Comment:
@saragluna , @haoozhang 
FYI: This fixed error about passing by value in for range.
Refs: https://go.dev/wiki/Range
> ![image](https://github.com/user-attachments/assets/2bc38735-6546-49b2-9f1d-8ee2c45ad9a4)